release_tools/README.adoc

```diff
@@ -21,24 +21,26 @@ authentication tokens for these services.
 * Your Jenkins user - your User ID is probably the same as your GitHub handle, to be sure,
   check your *profile page* in Jenkins.
 * A Jenkins token - generate a token on *Configure page* in Jenkins
+  While generating the token, make sure that your Jenkins user has build permissions in Jenkins.
 
-* Create `.env` file in `release_tools` directory, to hold your tokens:
+* Create `.env.yml` file in `release_tools` directory, to hold your tokens:
 +
-   GITHUB_TOKEN='<your github token here>'
-   JENKINS_USER='<you jenkins user ID>'
-   JENKINS_TOKEN='<you jenkins token here>'
+   github_token: '<your github token here>'
+   jenkins_user: '<you jenkins user ID>'
+   jenkins_token: '<you jenkins token here>'
```

Comment:
Just to be sure, the keys here are not case sensitive?

Reply:
I think they are not, this is derived from my current setup which I used during last release.

Reply:
Okay, thanks